internal/bminventory/inventory.go

```diff
@@ -482,8 +482,14 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 		if err != nil {
 			return nil, err
 		}
+		err = operators.EnsureLVMAndCNVNotEnabled(newOLMOperators)
```

Comment:
why we are not calling to updateOperatorsData from RegisterClusterInternal? it's code duplication

Reply:
Let's do that refactoring in another task?

Reply:
ok